source/voice_control/s3-box-customize.markdown

```diff
@@ -0,0 +1,189 @@
+- [Adding your images to the configuration](#to-add-your-images-to-the-configuration)
+  
+**Duration**: ca. 35 minutes, without counting the time to create the images.
+
+### About the images
+
+To make sure the images can be displayed properly on the S3-BOX screen, check out the following requirements.
+
+#### Illustrating 6 different states
+
+There are 6 different states to illustrate:
+
+- loading
+- idle
+- listening
+- thinking
+- replying
+- error
+
+#### Using light and dark image background
+
+To make it easier to distinguish the different states when you look at your screen, change the background depending on the state:
+
+- For loading and idle: use a dark background
+- For the listening, thinking, replying: use a light background
+- For error: As you like
+  
+You can also save the images with transparent background and then change the background in the configuration file.
+
+#### Image dimensions and file format
+
+- **Screen dimensions**: 320 x 240 pixels
+- **File format**: png
+
+### To prepare your own images
+
+1. Create your own images according the specifications defined [above](#about-the-images).
+2. Paste all 6 images into a folder. For example: `frenck`.
+3. Make sure you have [access to your configuration files](/common-tasks/os/#configuring-access-to-files).
+   - [Install the Samba add-on](/common-tasks/os/#installing-and-using-the-samba-add-on).
+   - Samba allows you to copy paste multiple files at once.
+4. Paste your image folder into the configuration folder:
+   - Using Samba, open the `config` folder and open the `ESPHome` folder.
+   - Paste your image folder in there.
+   ![ESP32-S3-BOX config file](/images/assist/s32-s3-add-image-folder.png)
+
+### To add your images to the configuration
+
+1. Go to **Settings** > **Add-ons**, select the **ESPHome** add-on and **Open Web UI**.
+2. On the **ESP32-S3-BOX** add-on, select edit.
+   - **Result**: An editor opens, showing the configuration file.
+   ![ESP32-S3-BOX config file](/images/assist/esp32-adopt-s3-01.png)
+
+3. For this tutorial, we will use some images of Frenck.
+   - To add your images, add the following lines into the `substitution` block.
+    {% raw %}
+
+     ```yaml
+      loading_illustration_file: /config/esphome/frenck/loading_320_240.png
+      idle_illustration_file: /config/esphome/frenck/idle_320_240.png
+      listening_illustration_file: /config/esphome/frenck/listening_320_240.png
+      thinking_illustration_file: /config/esphome/frenck/thinking_320_240.png
+      replying_illustration_file: /config/esphome/frenck/replying_320_240.png
+      error_illustration_file: /config/esphome/frenck/error_320_240.png
+      ```
+
+    {% endraw %}
+
+4. If you used transparency in your images and you want to change the background color, add the following lines:
+   - The default is black and white.
+   - If you want to use different colors, replace the corresponding hexadecimal color code.
```

Comment:
Should we link to a website where a user can understand what hexadecimal color codes are and click on colors to get them ?

Should we explain 000000 is black and ffffff is white ?

Reply:
The best is to ask them to look for "Color Picker" on Google.
They have a nice widget directly on the search page
![Uploading image.png…]()